main.go

```diff
@@ -58,6 +59,17 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	// Probably overkill given that a check exists the the controllers package,
```

Comment:
```suggestion
	// Probably overkill given that a check exists the controllers package,
```
Actually, shall we simply remove the entire comment because (1) there is no check in the controller package, (2) it's certainly not overkill and (3) it seems the most reasonable (the earliest) place to check the env?